product_secondary_unit/tests/test_product_second_unit.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import SavepointCase
+
+
+class TestProductSecondaryUnit(SavepointCase):
+    at_install = False
+    post_install = True
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.product_uom_kg = cls.env.ref("uom.product_uom_kgm")
+        cls.product_uom_unit = cls.env.ref("uom.product_uom_unit")
+        cls.product = cls.env["product.template"].create(
+            {
+                "name": "test",
+                "uom_id": cls.product_uom_kg.id,
+                "uom_po_id": cls.product_uom_kg.id,
+                "secondary_uom_ids": [
```

Comment:
Maintain format like as previous version...

Reply:
This is done automatically by black